tests/v2/validation/provisioning/provisioning_node_driver_test.go

```diff
@@ -335,8 +172,10 @@ func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning_RKE2AWSCluster()
 	}
 }
 
-func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning_RKE2AWSClusterDynamicInput() {
+func (r *RKE2NodeDriverProvisioningTestSuite) Provisioning_RKE2ClusterDynamicInput(provider Provider) {
+
 	nodesAndRoles := NodesAndRolesInput()
+	fmt.Printf("%v", nodesAndRoles)
```

Comment:
Either remove this print statement or make it a a log statement. Something like this ` r.T().Logf("Node Config: %v", nodesAndRoles) `